src/theme/index.ts

```diff
@@ -41,16 +42,47 @@ const fonts = {
 
 const theme = {
   colorsDeprecated,
+  colors,
   fonts,
   space,
   screens,
   radii,
 }
 
+const createTheme = ({
+  contrasts,
+  space: newSpace,
+  screens: newScreens,
+  radii: newRadii,
+  fonts: newFonts,
+}: {
+  contrasts?: Record<Color, Partial<ContrastType>>
+  space?: Spaces
+  screens?: ScreenSize
+  radii?: Record<string, string>
+  fonts?: Record<string, string>
+}) => ({
+  ...theme,
+  ...(contrasts ? { colors: colorsTokens(contrasts) } : undefined),
```

Comment:
You can't spread undefined :)

```suggestion
  ...(contrasts ? { colors: colorsTokens(contrasts) } : {}),
```

Reply:
Oh right my bad I changed it